build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -66,7 +61,6 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
```

Comment:
We also need to clean up the Compile Item, the EmbeddedResource item and move the reference to Microsoft.Net.Sdk to the item group where the other package references are.